### ISSUE TITLE
Data Type not properly parsed out for API Docs for SOAP WSDL

### ISSUE THREAD

**Issue by philicious** [dreamfactorysoftware/dreamfactory#58]
Example uses the `/api/v2_soap?wsdl=1` of Magento in WS-I compliance mode.

DF fails/(or just doesnt try) to parse the arguments for API Docs correctly.

WSDL function definition
```
<wsdl:message name="login">
  <wsdl:part name="parameters" element="typens:loginParam"/>
</wsdl:message>
```
and the parameters are a reference on
```
<xsd:element name="loginParam">
  <xsd:complexType><xsd:sequence>
    <xsd:element minOccurs="1" maxOccurs="1" name="username" type="xsd:string"/>
    <xsd:element minOccurs="1" maxOccurs="1" name="apiKey" type="xsd:string"/>
  </xsd:sequence></xsd:complexType>
</xsd:element>
```
However DF only shows `Data Type` as
```
loginParam {
  loginParam
}
```

**Comment by philicious** [comment]
just rechecked: this is still the case with DF 2.2 / df-soap 0.3.0

(however this might not be a bug but rather an enhancement)

**Comment by philicious** [comment]
this is fixed with the upcoming release following 2.2.1 / df-soap 0.3.1
